From: Dilshod Mukhtarov <dilshodm@gmail.com>
Date: Wed, 29 May 2024 12:30:00 +0400
Subject: [PATCH] Fix mbgl Mac build error

mbgl/util/unique_any.hpp:51:31: error: no member named 'move' in namespace 'std'

diff -Nru qtlocation/src/3rdparty/mapbox-gl-native/include/mbgl/util/unique_any.hpp qtlocation.new/src/3rdparty/mapbox-gl-native/include/mbgl/util/unique_any.hpp
--- a/qtlocation/src/3rdparty/mapbox-gl-native/include/mbgl/util/unique_any.hpp	2024-05-08 09:14:45.000000000 +0400
+++ b/qtlocation/src/3rdparty/mapbox-gl-native/include/mbgl/util/unique_any.hpp	2024-05-29 12:19:17.315123187 +0400
@@ -3,6 +3,7 @@
 #include <typeinfo>
 #include <type_traits>
 #include <stdexcept>
+#include <utility>
 namespace mbgl {
 namespace util {
 
